### ISSUE TITLE
errors on CentOS 7.6

### ISSUE THREAD

**Issue by azuercher** [ctomkow/swITch#4]
Hi Craig,
I'm trying to use your script to bulk edit cisco devices.  When I run it i get the following errors and I'm unsure how to fix.  Any ideas?

Thanks,
Aaron

> 
> ]# python3 ./swITch.py -va auth.txt -i ip.list -s set-clock.cmd
> Traceback (most recent call last):
>   File "./swITch.py", line 300, in <module>
>     swITch()
>   File "./swITch.py", line 84, in __init__
>     args.port, args.set, args.quiet, args.file, args.verbose, args.zomg)
>   File "./swITch.py", line 173, in main
>     if self.is_comment(self.strip_whitespace(raw_cmd)):
>   File "./swITch.py", line 293, in is_comment
>     if str[0] == '#':
> IndexError: string index out of range

> ]# python3 ./swITch.py -va auth.txt -i '10.1.1.246,cisco_ios' -c set-clock.cmd
> Traceback (most recent call last):
>   File "./swITch.py", line 300, in <module>
>     swITch()
>   File "./swITch.py", line 84, in __init__
>     args.port, args.set, args.quiet, args.file, args.verbose, args.zomg)
>   File "./swITch.py", line 165, in main
>     if self.is_comment(self.strip_whitespace(raw_cmd)):
>   File "./swITch.py", line 293, in is_comment
>     if str[0] == '#':
> IndexError: string index out of range

**Comment by ctomkow** [comment]
Hi Aaron. Sorry, I missed this issue you opened a while ago.

I figured out the issue. If there are empty lines in one of the config files, the script chokes when it checks for a leading '#'. This was a regression I introduced when I added the comment checking.

e.g. this will fail
```
show ver

show interface status
```

I pushed a quick fix for the issue.
https://github.com/ctomkow/swITch/commit/4344601490427c997078410b460a5b3ad593ed46

Thanks.
